manifests/cargo-rdme.json

```diff
@@ -21,6 +21,16 @@
   "1.4": {
     "version": "1.4.2"
   },
```

Comment:
The three "versions" here weren't updated.  Is this intentional, or was there some bug in the manifest generation?

Reply:
Hmm. That's annoying, but understandable. I was going to suggest that the install action should warn if you're in this state, but I wouldn't have seen the place where that would have been reported anyway 😅. That is, for all I know, the warning already exists.

I can't see where this is tracked upstream? Cc @orium

Reply:
> I can't see where this is tracked upstream?

I guess it is https://github.com/orium/cargo-rdme/issues/239 / https://github.com/rust-build/rust-build.action/pull/99 (Linux, fixed), https://github.com/rust-build/rust-build.action/issues/88 (macOS)

A practical action on my part for projects with problematic release workflows is to help them switch to an action that is understandable and easy for me to set up and actually used in various parts of the ecosystem, such as in the tarpaulin case (https://github.com/taiki-e/install-action/pull/42#issuecomment-1372184541, https://github.com/xd009642/tarpaulin/pull/1174).